ServiceRequest/ServiceRequest-WGSTestOrderForm-TrioTestingProband-Example.json

```diff
@@ -112,15 +116,19 @@
       "reference": "Observation/Observation-NoTransfusion-Example"
```

Comment:
Not a blocker but the parents also have Bone marrow transplant, transfusions and consanguinous union observations, would be good to add observations that link to the mother and father patient resources for completeness

Reply:
I am not sure how the parent consanguinous union observations will link with the FMH resource, so I have only created the observations.

Reply:
You can do this similarly to the FMH resource for the proband, just replace the patient reference with patient references for the father and mother) for the 2 FMH resources created to capture their family history.

Reply:
Also to make it clear that the observations relate to the FMH, could you also add an extension to the FMH resources which link to the consanguinous union observation: https://hl7.org/fhir/r4/extension-family-member-history-genetics-observation.html

Reply:
Sorted as discussed.